Makefile

```diff
@@ -80,12 +81,24 @@ TESTS = \
 	dcurl \
 	pow
 
+ifeq ("$(BUILD_COMPAT)", "1")
+TESTS += ccurl-multi_pow
+endif
+
 TESTS := $(addprefix $(OUT)/test-, $(TESTS))
 
+TRIALS = \
```

Comment:
Again, I dislike the operations you pre-define. Both `new-task` and `get-result` are not format terms we shall introduce. Be careful about the naming scheme.